serverless/pages/search-your-data-semantic-search.asciidoc

```diff
@@ -6,140 +6,50 @@
 
 preview:[]
 
-Semantic search is a search method that helps you find data based on the intent
-and contextual meaning of a search query, instead of a match on query terms
-(lexical search).
+Semantic search is a search method that helps you find data based on the intent and contextual meaning of a search query, instead of a match on query terms (lexical search).
 
-{es} provides semantic search capabilities using {ml-docs}/ml-nlp.html[natural
-language processing (NLP)] and vector search. Deploying an NLP model to {es}
-enables it to extract text embeddings out of text. Embeddings are vectors that
-provide a numeric representation of a text. Pieces of content with similar
-meaning have similar representations.
+Elasticsearch provides various semantic search capabilities using natural language processing (NLP) and vector search. Using an NLP model enables you to extract text embeddings out of text. Embeddings are vectors that provide a numeric representation of a text. Pieces of content with similar meaning have similar representations.
```

Comment:
Let's add a sentence here that states that there are 3 main workflows for implementing semantic search with {es}, with each one being progressively more complex.

We should maintain the ordering in the sections:

1. `semantic_text` workflow
1. Inference API workflow
1. Model deployment workflow

Note that you've changed the ordering and number 3 should point [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/semantic-search-deployed-nlp-model.html)